src/framework/parsers/draco-decoder.js

```diff
@@ -78,6 +101,44 @@ class JobQueue {
     }
 }
 
+const downloadScript = (url) => {
+    return new Promise((resolve, reject) => {
+        const options = {
+            cache: true,
+            responseType: 'text',
+            retry: downloadMaxRetries > 0,
+            maxRetries: downloadMaxRetries
+        };
+
+        http.get(url, options, (err, response) => {
+            if (err) {
+                reject(err);
+            } else {
+                resolve(response);
+            }
+        });
+    });
```

Comment:
Just throwing it out there, but any reason not to start using `fetch` in the engine codebase?

Reply:
https://caniuse.com/?search=fetch

Reply:
This was left to leverage http retry logic.